modules/sdk/project-templates-spring-mvc-portlet/src/main/resources/archetype-resources/build.gradle

```diff
@@ -0,0 +1,49 @@
+buildscript {
+    repositories {
+        mavenLocal()
+        maven {
+            url "https://cdn.lfrs.sl/repository.liferay.com/nexus/content/groups/public"
+        }
+    }
+
+    dependencies {
+        classpath group: "com.liferay", name: "com.liferay.gradle.plugins.css.builder", version: "latest.release"
```

Comment:
use this instead
`@com.liferay.gradle.plugins.css.builder.version@`
and
`@com.liferay.css.builder.version@`
instead, and you can specify those in `modules/build.gradle` file.